hieradata/class/ci_master.yaml

```diff
@@ -0,0 +1,81 @@
+---
+
+govuk_ci::master::plugin_hash:
```

Comment:
I still can't make up mind whether it's nicer to do like this or just call `govuk_jenkins::plugins` directly